src/lib/plugin.js

```diff
@@ -428,20 +428,24 @@ class FiveBellsLedger extends EventEmitter2 {
     if (message.ledger !== this.ledgerContext.prefix) {
       throw new errors.InvalidFieldsError('invalid ledger')
     }
-    if (typeof message.to !== 'string' && typeof message.account !== 'string') {
-      throw new errors.InvalidFieldsError('invalid account')
-    }
-    if (message.to && message.account) {
-      throw new errors.InvalidFieldsError('"to" and "account" cannot both be defined')
+    if (typeof message.to !== 'string') {
+      // check for deprecated Message format, from before https://github.com/interledger/rfcs/commit/61958f54c268e5a52e1b85f090df02646b0dda38
+      if (typeof message.account === 'string') {
+        util.deprecate(() => {}, 'switch from using "account" to "to"')()
+        const correctedMessage = Object.assign({}, message)
+        correctedMessage.to = message.account
+        delete correctedMessage.account
+        yield * this._sendMessage(correctedMessage)
```

Comment:
This should just be `yield` instead of `yield  *`, if you want to await `_sendMessage`